samples/DotNet/Azure.Messaging.ServiceBus/BasicSendReceiveTutorialwithFilters/BasicSendReceiveTutorialWithFilters/Extensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+//using Microsoft.Azure.ServiceBus;
+using Newtonsoft.Json;
+using Azure.Messaging.ServiceBus;
+using System;
+
+namespace BasicSendReceiveTutorialWithFilters
+{
+    public static class Extensions
+    {
+        public static T As<T>(this ServiceBusReceivedMessage message) where T : class
+        {
+            return JsonConvert.DeserializeObject<T>(Encoding.UTF8.GetString(message.Body.ToBytes().ToArray()));
```

Comment:
Any reason why [`System.Text.Json`](https://docs.microsoft.com/en-us/dotnet/api/system.text.json) is not used?